src/ray/raylet/worker_pool_test.cc

```diff
@@ -1422,6 +1437,7 @@ TEST_F(WorkerPoolTest, PopWorkerStatus) {
 
   /* Test PopWorkerStatus RuntimeEnvCreationFailed */
   // Create a task without push worker.
+  // Fault To be fixed
```

Comment:
The below line is taking too long. Will fix this.

Reply:
Fixed. This was due to inconsistencies in the value of  `startup_token_`(gets updated just after calling `MonitorStartingWorkerProcess`) inside lambda function being sent to `async_wait` as a parameter.